etc/deprecated.csv

```diff
@@ -22,5 +22,9 @@ T:System.Net.HttpWebRequest,System.Net,HttpWebRequest,,DE0003
 T:System.Net.WebClient,System.Net,WebClient,,DE0004
 T:System.Net.WebRequest,System.Net,WebRequest,,DE0003
 T:System.Net.Mail.SmtpClient,System.Net.Mail,SmtpClient,,DE0005
+M:System.Reflection.IntrospectionExtensions.GetTypeInfo(System.Type),System.Reflection,IntrospectionExtensions,GetTypeInfo(Type),DE0008
```

Comment:
Is this enforced via analyzers?

Reply:
Yes. The analyzer uses the CSV file to produce a [database ("API store")](https://github.com/dotnet/platform-compat/blob/master/src/PlatformCompat.Analyzers/Store/ApiStore.cs#L41) to match symbols against it.